public/docs/slide-example.md

```diff
@@ -80,4 +122,142 @@ default/fast/slow
 
 ---
 
-# The End
\ No newline at end of file
+## Themes
+
+reveal.js comes with a few themes built in:  
+Black (default) - White - League - Sky - Beige - Simple  
+Serif - Blood - Night - Moon - Solarized
+
+It can be set in Yaml slideOptions
+
+---
+
+<!-- .slide: data-background="#1A237E" -->
+
+`<!-- .slide: data-background="#1A237E" -->`
+
+Is the background syntax
+
+---
+
+<!-- .slide: data-background="https://s3.amazonaws.com/hakim-static/reveal-js/image-placeholder.png" data-background-color="#005" -->
+
+<div style="color: #fff;">
+
+## Image Backgrounds
+
+`<!-- .slide: data-background="image.png"-->`
+
+</div>
+
+----
+
+<!-- .slide: data-background="https://s3.amazonaws.com/hakim-static/reveal-js/image-placeholder.png" data-background-repeat="repeat" data-background-size="100px" data-background-color="#005" -->
+
+<div style="color: #fff;">
+
+## Tiled Backgrounds
+
+`<!-- .slide: data-background="image.png" data-background-repeat="repeat" data-background-size="100px" -->`
+
+</div>
+
+----
+
+<!-- .slide: data-background-video="https://s3.amazonaws.com/static.slid.es/site/homepage/v1/homepage-video-editor.mp4,https://s3.amazonaws.com/static.slid.es/site/homepage/v1/homepage-video-editor.webm" data-background-color="#000000" -->
+
+<div style="background-color: rgba(0, 0, 0, 0.9); color: #fff; padding: 20px;">
+
+## Video Backgrounds
+
+`<!-- .slide: data-background-video="video.mp4,video.webm" -->`
+
+</div>
+
+----
+
+<!-- .slide: data-background="http://i.giphy.com/90F8aUepslB84.gif" -->
+
+## ... and GIFs!
+
+---
+
+## Pretty Code
+
+``` javascript
+function linkify( selector ) {
+  if( supports3DTransforms ) {
+
+    var nodes = document.querySelectorAll( selector );
+
+    for( var i = 0, len = nodes.length; i < len; i++ ) {
+      var node = nodes[i];
+
+      if( !node.className ) {
+        node.className += ' roll';
+      }
+    }
+  }
+}
+```
+Code syntax highlighting courtesy of [highlight.js](http://softwaremaniacs.org/soft/highlight/en/description/).
+
+---
+
+## Marvelous List
+
+*   No order here
+*   Or here
+*   Or here
+*   Or here
+
+---
+
+## Fantastic Ordered List
+
+1.  One is smaller than...
+2.  Two is smaller than...
+3.  Three!
+
+---
+
+## Tabular Tables
+
+| Item | Value | Quantity |
+| ---- | ----- | -------- |
+| Apples | $1 | 7 |
+| Lemonade | $2 | 18 |
+| Bread | $3 | 2 |
+
+---
+
+## Clever Quotes
+
+> “For years there has been a theory that millions of monkeys typing at random on millions of typewriters would reproduce the entire works of Shakespeare. The Internet has proven this theory to be untrue.”
+
+---
+
+## Intergalactic Interconnections
+
+You can link between slides internally, [like this](#/1/3).
+
+---
+
+## Speaker View
```

Comment:
As the official feature is called "Speaker Notes" I would also call this section "Speaker Notes". It makes it easier for the people who already use `reveal.js` to stay on topic.